kitsune/questions/jinja2/questions/question_list.html

```diff
@@ -63,41 +63,43 @@
         <h1 class="sumo-page-heading {% if not logo %}no-logo{% endif %}">
           <span class="product-title-text">{{ title }}</span>
         </h1>
-        <div id="recent-stats">
-          <div class="no-reply forum--masthead-text">
-            <p class="page-heading--intro-text">
-              {% if recent_unanswered_count > 0 %}
-                {% if request.user.is_authenticated %}
+        {% if not topic_navigation %}
+          <div id="recent-stats">
+            <div class="no-reply forum--masthead-text">
+              <p class="page-heading--intro-text">
+                {% if recent_unanswered_count > 0 %}
+                  {% if request.user.is_authenticated %}
+                    {% trans recent_unanswered_count, url=questions_url(filter='recently-unanswered', show='all') %}
+                      {{recent_unanswered_count}} question in the last 24 hours has no reply.
+                      <a href="{{url}}">Help solve it!</a>
+                      {% pluralize recent_unanswered_count %}
```

Comment:
I never knew about the use of `{% pluralize %}` within a `trans` statement until now! Nice!